Tasks/Maven/maventask.ts

```diff
@@ -40,6 +41,19 @@ else {
         mvnExec = tl.which('mvn', true);
     }
 }
+
+// On Windows, append .cmd or .bat to the executable as necessary
+if (os.type().match(/^Win/)) {
+    if (tl.exist(mvnExec + '.cmd')) {
```

Comment:
Shouldn't we check first that mvnExec does not have extension already? It looks like we might create pathes ending with mvn.cmd.bat, mvn.cmd.cmd, etc